### ISSUE TITLE
Mass query import for Apollo projects

### ISSUE THREAD

**Issue by DmitryTsepelev** [DmitryTsepelev/graphql-ruby-persisted_queries#5]

**Comment by JanStevens** [comment]
Maybe you could use: https://graphql-ruby.org/javascript_client/sync and store it to a file and then on load read from that file and populate the cache?

This way you could have a CI setup that commits the result to a file which wile be loaded when your app starts to seed the store

**Comment by DmitryTsepelev** [comment]
Good point, pre–warmed cache might be helpful in case of unreliable network
